composeApp/src/commonMain/kotlin/com/majotyler/hiittimer/presentation/common/navigation/NavigationRoot.kt

```diff
@@ -16,10 +17,16 @@ import com.majotyler.hiittimer.presentation.homeScreen.HomeScreen
 import com.majotyler.hiittimer.presentation.homeScreen.HomeViewModel
 import kotlinx.serialization.modules.SerializersModule
 import kotlinx.serialization.modules.polymorphic
-import kotlinx.serialization.serializer
 
 @Composable
 fun NavigationRoot() {
+    HiitAppTheme {
+        HiitNavDisplay()
+    }
```

Comment:
To apply the Theme, we are in a `Composable`, so all I have to do is nest the Composables of our app (including navigation!) in the theme.

Reply:
The Theme provides what we need when we access colors through `MaterialTheme.colorScheme` - that is why that is better than specifying colors in code.